packages/p2p-v2/src/routes/Home/index.tsx

```diff
@@ -16,9 +17,11 @@ const Home: React.FC<{ path: string }> = ({ path }) => {
 
     return (
         <>
-            {isMobile && <MobileCloseHeader />}
+            {/* {isMobile && <MobileCloseHeader />} */}
+            <MobileCloseHeader />
             <div className='p2p-v2-home-style'>
-                <div> Welcome to P2P Version 2 from {path} Page ;) </div>
+                {/* <div> Welcome to P2P Version 2 from {path} Page ;) </div> */}
+                <MyProfileContent />
```

Comment:
i dont think we need to push these changes right 👀

Reply:
yea these are test changes, I've removed them